base/linalg/triangular.jl

```diff
@@ -1900,14 +1907,10 @@ end
 function sqrtm{T}(A::UnitUpperTriangular{T})
```

Comment:
Since `UnitUpperTriangular{T}` is not a subtype of `UpperTriangular{T}` (??), I think it would be

    sqrtm(A::UnitUpperTriangular) = UnitUpperTriangular(sqrtm(UpperTriangular(A),Val{true}))

For some reason this is *faster* than the present method, which confuses me. This version involves two additional conversions and some unnecessary arithmetic operations in the loop (of order O(n) as you mention)! Are some ops on `UnitUpperTriangular` slower than the same op on `UpperTriangular`?

Will make this change later unless someone can enlighten me